sql/mysql/sqlspec_test.go

```diff
@@ -331,6 +331,13 @@ func TestMarshalSpecColumnType(t *testing.T) {
 			},
 			expected: specutil.NewCol("column", "int8"),
 		},
+		{
+			schem: &schema.IntegerType{
+				T:        tMediumInt,
+				Unsigned: false,
+			},
+			expected: specutil.NewCol("column", "int24"),
```

Comment:
No such virtual type "uint24", this is a specific mysql int size that doesn't have a corresponding setting on other dbs so we chose to omit it

Reply:
You can't omit this type it's used, for example:
```mysql://rfamro@tcp(mysql-rfam-public.ebi.ac.uk:4497)/Rfam```
without supporting this type we won't be able to inspect any db using it.

Reply:
let me clarify, the Atlas DDL has the concept of [Virtual Types](https://atlasgo.io/sql-hcl#virtual-types), it has a list of types that a user can use and Atlas will project into the db specific type. We chose not to include `uint24` in this list. Instead a user can always use the database-specific type name. In this case it would be  `mediumint`. 

The bug is actually here:
https://sourcegraph.com/github.com/ariga/atlas@master/-/blob/sql/mysql/sqlspec.go?L330-331

the type of the column should be `mediumint`.